### DISCUSSION TITLE
Strange error I don't know how to deal

### DISCUSSION THREAD

**Discussion by Aledosim** [neoclide/coc.nvim#4428] [Q&A] [answered]
Hello,
When I launch vim, I got this message and COC don't work. I already searched on the internet and I didnt find any clue about it.
It is in portuguese, basically it tells that bufadd is an unkown function and there are fell endifs missing, but the endifs are there in the code and I don't know with what this function is related.
My vim version is 8.1 and nodejs is 18.12.1
I apreciate if someone could point where I can get help too, I don't know if this is the right place.
```
Erro detectado ao processar function coc#rpc#start_server[57]..coc#util#get_data_home[18]..coc#notify#create[87]..coc#float#create_float_win[4]..coc#float#create_buf:
linha    5:
E117: Função desconhecida: bufadd
Erro detectado ao processar function coc#rpc#start_server[57]..coc#util#get_data_home:
linha   18:
E171: :endif faltando
Erro detectado ao processar function coc#rpc#start_server:
linha   57:
E171: :endif faltando
Erro detectado ao processar /home/user/.vim_runtime/sources_non_forked/coc.nvim/plugin/coc.vim:
linha   56:
E171: :endif faltando
Aperte ENTER ou digite um comando para continuar
```

**Comment by chemzqm** [comment, marked as answer]
Seems you have vim < 8.1.1719 which is not supported

**Reply by Aledosim** [reply]
I think you're right. In my office it works (vim version 8.2) and it crashed in my home after I updated it.
Thanks
